### PR TITLE
Addition of redis_configs parameter, allows additional configuration for Redis

### PR DESCRIPTION
**Pull request by sumeetoc**

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -72,6 +72,12 @@ variable "redis_version" {
   default     = null
 }
 
+variable "redis_configs" {
+  description = "The Redis configuration parameters."
```
Comment (bharathkkb) [inline variables.tf:76]: nit: Can we add a reference to where a user can go to find [additional configs](https://www.terraform.io/docs/providers/google/d/redis_instance.html#redis_configs)? Perhaps similar to [this]( https://github.com/terraform-google-modules/terraform-google-sql-db/blob/76244bdeed4896dbad645345b11c71df2ddfc87a/modules/mysql/variables.tf#L119)

Comment (sumeetoc) [reply variables.tf]: > nit: Can we add a reference to where a user can go to find [additional configs](https://www.terraform.io/docs/providers/google/d/redis_instance.html#redis_configs)? Perhaps similar to [this](https://github.com/terraform-google-modules/terraform-google-sql-db/blob/76244bdeed4896dbad645345b11c71df2ddfc87a/modules/mysql/variables.tf#L119)

Done

**Comment by sumeetoc** [comment]
Hi @bharathkkb,

Any estimates on when this PR would be merged in master for us to use the added feature?

Thanks!

**Comment by bharathkkb** [comment]
This should be good to go. Thanks for the doc update!
/cc @morgante

**Pull request merged**